src/AggregateFunctions/Combinators/AggregateFunctionResample.cpp

```diff
@@ -47,6 +47,8 @@ class AggregateFunctionCombinatorResample final : public IAggregateFunctionCombi
     {
         WhichDataType which{arguments.back()};
 
+        chassert(params.size() > 3);
```

Comment:
chassert will not save us in production 
```suggestion
        if (params.size() < 3)
            throw Exception(ErrorCodes::NUMBER_OF_ARGUMENTS_DOESNT_MATCH,
                    "Incorrect number of parameters for aggregate function with {} suffix", getName());
```

Reply:
Yes I know, I wanted to see if it reproduces in debug builds with our usual test suite. Adding an `if` works better, I agree.